### ISSUE TITLE
auto-friending when running extension 

### ISSUE THREAD

**Issue by onebeastchris** [rtm516/MCXboxBroadcast#20]
Hey, I recently noticed that the bot doesn't re-add you automatically so I have to accept friend requests manually on the account first. Would it be possible for the extension to do so automatically?
thanks in advance!

**Comment by PizzaThatCodes** [comment]
Added a fix for this, just waiting on it being merged #21

**Comment by onebeastchris** [comment]
dang, sick! thanks :)

**Comment by PizzaThatCodes** [comment]
If you want to use it now since the merge hasn't happened yet, you can get the .jar on my repository for it

https://github.com/PizzaThatCodes/MCXboxBroadcast/releases/tag/release

**Comment by onebeastchris** [comment]
oops, duplicate of #16 - my bad
